pkg/roachprod/fluentbit/fluentbit.go

```diff
@@ -0,0 +1,104 @@
+package fluentbit
+
+import (
+	"bytes"
+	"context"
+	_ "embed"
+	"strings"
+	"text/template"
+
+	"github.com/cockroachdb/cockroach/pkg/roachprod/install"
+	"github.com/cockroachdb/cockroach/pkg/roachprod/logger"
+	"github.com/cockroachdb/errors"
+)
+
+//go:embed files/fluent-bit.yaml.tmpl
+var fluentBitTemplate string
+
+//go:embed files/fluent-bit.service
+var fluentBitSystemdUnit string
+
+type Config struct {
```

Comment:
I’m guessing the linter will say this too, but we always comment exported types and functions.